### PR TITLE
Cleaning repo with new SCT command 

### PR DESCRIPTION
**Pull request by plbenveniste**
Creating this short PR to add the instructions to use the model using SCT. 

The script to perform inference was moved to the utils folder so that it is still easy to perform prediction if the person has trained its own model. 
Both README.md were updated accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,63 +29,37 @@ For the packaging we decided to keep only fold 4 as it has the best dice score a
...
 </details>
 
-For information on how to retrain the same model, refer to this file [README_training_model.md](https://github.com/ivadomed/model_seg_mouse-sc_wm-gm_t1/blob/main/utils/README_training_model.md). 
+For information on how to retrain the same model, refer to this file [README.md](./utils/README.md). 
 
-If you wish to try the model on your own data, follow the instructions at [Installation](#installation) and [Perform predictions](#perform-predictions).
+## How to use the model
 
-## Installation
+This is the recommended method to use the model (for other methode refer to [README.md](./utils/README.md)).
```
[the remark this answers is not in the document]
Comment (plbenveniste) [reply README.md:36]: True ! It's mostly about training. It also includes the inference script, though. Maybe `training_inference_scripts` is better then ?

[the remark this answers is not in the document]
Comment (valosekj) [reply README.md:36]: Do we still need the inference script, as the model is now packaged in SCTv6.2? For example, for SCIseg, we [completely removed](https://github.com/ivadomed/model_seg_sci/pull/81) all "old" inference scripts and instructions to keep the repo light and clear. Even after removing the script from the main branch, the script will still be available in the git history.

[the remark this answers is not in the document]
Comment (plbenveniste) [reply README.md:36]: Yes ! It's a good idea.

[the remark this answers is not in the document]
Comment (plbenveniste) [reply README.md]: Done !

**Review by valosekj — APPROVED** [review]
LGTM, thanks!

**Pull request merged**